product-service/handlers/getProductsList.ts

```diff
@@ -0,0 +1,25 @@
+import {Context, APIGatewayEvent, APIGatewayProxyResult} from "aws-lambda";
+import {getAllProductsWithDelay} from "service/products";
+
+export async function getProductsList(
+    event: APIGatewayEvent,
+    context: Context
+): Promise<APIGatewayProxyResult> {
+    try {
+        return {
+            statusCode: 200,
+            headers: {
+                "Access-Control-Allow-Origin": "*"
+            },
+            body: JSON.stringify(await getAllProductsWithDelay(40)),
+        };
+    } catch (e) {
+        return {
+            statusCode: 200,
+            headers: {
+                "Access-Control-Allow-Origin": "*"
+            },
+            body: "Products not found",
```

Comment:
The same. Think about generating internal server error in catch part (status code 500)

Reply:
statusCode 500 shall be implemented in Task 4.
https://github.com/rolling-scopes-school/nodejs-aws-tasks/blob/main/task4-rds/task.md
"All lambdas return error 500 status code on any error (DB connection, any unhandled error in code)"